src/ArgentPonyWarcraftClient/IsExternalInit.cs

```diff
@@ -0,0 +1,10 @@
+﻿#if NETSTANDARD2_0
+
+namespace System.Runtime.CompilerServices
+{
+    // This is required to enable the init keyword while targeting lower than .NET 5
+    // See https://stackoverflow.com/a/64749403/342581.
```

Comment:
Thanks for the comment on this! I remember trying to use `init` properties in a .NET Std 2.0 library and thought it was only for .NET 5. I'll definitely keep this in mind for my own work!